disnake/webhook/sync.py

```diff
@@ -565,6 +565,11 @@ class SyncWebhook(BaseWebhook):
         Only given if :attr:`type` is :attr:`WebhookType.channel_follower`.
 
         .. versionadded:: 2.0
+
+    application_id: Optional[:class:`int`]
```

Comment:
In what case is this ever filled?

Reply:
I suppose it's kind of an edge case, but `SyncWebhook.from_url(...).fetch()` would be one possible case where this is set, assuming the url is for an application-owned webhook